server.js

```diff
@@ -16,6 +16,7 @@ const app = express()
 const PORT = process.env.PORT ?? 3000
 const TIMEOUT = process.env.TIMEOUT ?? 2000
 const SERVER_NAME = process.env.SERVER_NAME
+const PRESERVE_STORAGE = process.env.PRESERVE_STORAGE === 'true' || false;
```

Comment:
To maintain backward compatibility, I would like to change the default value so that the current behavior is preserved by default.
```suggestion
const PRESERVE_STORAGE = (process.env.PRESERVE_STORAGE ?? 'true') === 'true';
```